game/assets/game_text.txt

```diff
@@ -0,0 +1,11 @@
+;; "project file" for text make tool.
```

Comment:
a suggestion - can we avoid naming files with LISP syntax code/data in them `*.txt`?  I find this personally confusing and means they cannot be syntax highlighted ontop of that.  Perhaps this is a good use-case for the `.gd` extension instead? (GOAL Data?)

Reply:
I agree. I actually named one of them "*.gc" when messing with stuff but ended up deleting it. I'll rename them now.